python/src/main/python/drivers/runspv.py

```diff
@@ -231,13 +237,24 @@ def spirvopt_path():
     return tool_on_path('spirv-opt')
 
 
-def maybe_add_catchsegv(cmd: List[str]) -> None:
-    # Add catchsegv to cmd if it is available
-    try:
-        cmd.append(tool_on_path('catchsegv'))
-    except ToolNotOnPathError:
-        # Didn't find catchsegv on path; that's OK
-        pass
+# FIXME: using gdb wrapping leads to a weird issue where, once a test times out,
+# subsequent tests leads to the whole worker being SIGSTOP'ed.
+# See https://github.com/google/graphicsfuzz/issues/338
+
+# def maybe_add_gdb(cmd: List[str]) -> None:
+#     # If gdb is available, use it to obtain backtraces in case of segfault
+
+#     # TODO: using gdb as a wrapper adds at least a second-long overhead to each
```

Comment:
Can you change TODO to TODO(issue number) where 'issue number' is the number of the associated issue for this optimization idea?

Reply:
(I like the reference to the issue above for the FIXME.  But this sound like it deserves a different issue.)

Reply:
I propose we make this PR:

* Remove use of `catchsegv`
* Fix issue with subprocess_helper timeout

And we don't commit the commented out code; we instead create an issue, and put the code there or leave the code in a branch.

Reply:
See #349 for the fix of timeout.

Reply:
Looks good. Let's make this just:

* Remove use of `catchsegv`